cms/admin/pageadmin.py

```diff
@@ -740,11 +740,7 @@ def get_object(self, request, object_id, from_field=None):
         key is used if no field is provided. Return ``None`` if no match is
         found or the object_id fails validation.
         """
-        obj = super().get_object(request, object_id, from_field)
-
-        if obj:
-            obj.page.admin_content_cache[obj.language] = obj
-        return obj
+        return super().get_object(request, object_id, from_field)
```

Comment:
I think we can delete the whole method; all it does is calling `super()`.

Reply:
Hehe, absolutely. Hadn't had my coffee before removing the lines. Going to change it real quick...

Reply:
@fsbraun done :)